apps/obs-web/src/views/DebrieferErrors.vue

```diff
@@ -3,79 +3,133 @@
-    }
-  ];
-
-  private openNewDebriefingTab() {
-    const route = '/observer-web/debriefer/qa';
-    window.open(route, '_blank');
+import { createComponent, ref, onMounted, watch, computed } from '@vue/composition-api';
+import moment from 'moment';
+
+import { couchService } from '@boatnet/bn-couch';
+import { Client } from 'davenport';
+import { NoSubstitutionTemplateLiteral } from 'typescript';
+
+export default createComponent({
+  props: {
+    showData: Boolean,
+    isFullSize: Boolean
+  },
+  setup(props, context) {
+    const store = context.root.$store;
+    const state = store.state;
+    const masterDB: Client<any> = couchService.masterDB;
+
+    const errorColumns = [
+      { field: 'severity', header: 'Severity', key: 'errorSeverity', width: '80' },
+      { field: 'description', header: 'Description', key: 'errorDescription', width: '80'},
+      { field: 'tripNum', header: 'Trip #', key: 'errorTripNum', width: '80' },
+      { field: 'dateCreated', header: 'Date Created', key: 'dateErrorCreated', width: '80' },
+      { field: 'observer', header: 'Observer', key: 'errorObserver', width: '80' },
+      { field: 'status', header: 'Status', type: 'toggle',
+                listType: 'template',
+                list: ['Unresolved', 'Resolved'],
+                key: 'errorStatus',
+                isEditable: true,
+                width: '80'
+              },
+      { field: 'dateFixed', header: 'Date Fixed', key: 'dateErrorFixed', width: '80' },
+      { field: 'note', header: 'Note', type: 'textArea', key: 'errorNote', isEditable: true, width: '80' }
+    ];
+
+    const loading: any = ref(false);
+    const errorRows: any = ref([]);
+    const errorDocs: any = ref([]);
+
+    const openNewDebriefingTab = () => {
+      const route = '/observer-web/debriefer/qa';
+      window.open(route, '_blank');
+    };
+
+    const getErrors = async (tripId: number) => {
+      try {
+        const errorsQuery: any = await masterDB.view(
+          'obs_web',
+          'wcgop-trip-errors',
+          {reduce: false, include_docs: true, key: tripId} as any
+        );
+
+        const errors: any = [];
+
+        if (errorsQuery.rows.length > 0) {
+          errorDocs.value.push.apply(errorDocs.value, errorsQuery.rows);
+          for (const error of errorsQuery.rows[0].doc.errors) {
+            error.tripNum = errorsQuery.rows[0].doc.tripNumber;
+            error.note = error.notes;
+            error.uid = errorsQuery.rows[0].doc._id + '_' + errorsQuery.rows[0].doc.errors.indexOf(error);
+            error._rev = errorsQuery.rows[0].doc._rev;
+            errors.push(error);
+          }
+          errorRows.value.push.apply(errorRows.value, errors);
+        }
+      } catch (err) {
+        console.error(err);
+      }
+    };
+
+
+    const getTripErrors = () => {
+      loading.value = true;
+      errorDocs.value.length = 0;
+      errorRows.value = [];
+      for (const trip of state.debriefer.trips) {
```

Comment:
was thinking we could just show all the errors for an evaluation period rather than requiring them to select trips first since that's how the trips page works. But can probably decide that in the debriefer meeting, just wanted to make a note here.